_includes/plugins/tinymcespellchecker_content_langs.md

```diff
@@ -0,0 +1,50 @@
+### `content_langs`
+{% include configuration/content_langs_base.md %}
+
+> **Note:** The `content_langs` option replaces the `spellchecker_select_languages` option
+
+**Type:** `Array`
+
+**Default Value:**
+
+```js
+[
+  { title: 'English', code: 'en' },
+  { title: 'Spanish', code: 'es' },
+  { title: 'French', code: 'fr' },
+  { title: 'German', code: 'de' },
+  { title: 'Portuguese', code: 'pt' },
+  { title: 'Chinese', code: 'zh' }
+]
+```
+
+> **Note:** This default is provided by the Spell Checker Pro plugin, it is not available in the core editor. The languages specified by the deprecated [`spellchecker_select_language`](#spellchecker_select_language) option are used as a fallback if they are available.
```

Comment:
The previous note I left about this hasn't been resolved. To save finding it in this mess of comments, here it was:
> This note still seems out of place. I'm wondering if it should be above the Type as well? If we do move it, it'll need some rewording as well.

Reply:
Because it relates directly to the defaults, I thought it would make more sense putting it next to the defaults — also the notes at the top are getting really crowded.

Reply:
Fixed as discussed